Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gcr.io/dry-dock/u14:master.5
+ADD . /tmp
+RUN /tmp/script.sh && rm -rf /tmp
+CMD ["/bin/bash"]
```

Comment:
Also, would you please add a line break after each command? Let's keep everything consistent with what we have done with the base images